lib/capybara/node/finders.rb

```diff
@@ -57,15 +57,15 @@ def find(*args)
       #   @macro waiting_behavior
       #
       #
-      #   @option options [Boolean] checked  Match checked field?
-      #   @option options [Boolean] unchecked   Match unchecked field?
+      #   @option options [Boolean] checked       Match checked field?
+      #   @option options [Boolean] unchecked     Match unchecked field?
       #   @option options [Boolean, Symbol] disabled (false)  Match disabled field?
       #                                                       * true - only finds a disabled field
       #                                                       * false - only finds an enabled field
       #                                                       * :all - finds either an enabled or disabled field
-      #   @option options [Boolean] readonly Match readonly field?
-      #   @option options [String] with   Value of field to match on
-      #   @option options [String] type   Type of field to match on
+      #   @option options [Boolean] readonly      Match readonly field?
+      #   @option options [String/Regexp] with    Value of field to match on
```

Comment:
In Yard a types specifier list is a comma separated list of types, not / separated